install_config/install/docker_registry.adoc

```diff
@@ -545,6 +545,71 @@ Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
 
+== Advanced Configuration
+
+To enable managing the registry configuration file directly it is recommended that the configuration file be mounted as a secret volume.
+
+. link:#deploy-registry[Deploy the registry].
+. Edit the configuration file locally as needed. See details on
+https://github.com/docker/distribution/blob/master/docs/configuration.md#override-configuration-options[overriding registry options]. The initial file may be found in the link:https://github.com/openshift/origin/blob/master/images/dockerregistry/config.yml[Docker image registry source repository].
```

Comment:
Looks like you could link here instead for the registry override doc:

https://docs.docker.com/registry/configuration/

For the `config.yml` file, it looks short enough that maybe we could just inline it here instead of linking to the upstream code repo? Looking at the file's history, it looks like it hasn't had too much action (at least so far), as far as concerns of an in-line version here getting out of date.